portal-impl/src/portal.properties

```diff
@@ -131,7 +131,8 @@
         com.liferay.portal.upgrade.UpgradeProcess_6_1_1\,\
         com.liferay.portal.upgrade.UpgradeProcess_6_2_0\,\
         com.liferay.portal.upgrade.UpgradeProcess_7_0_0\,\
-        com.liferay.portal.upgrade.UpgradeProcess_7_0_1
+        com.liferay.portal.upgrade.UpgradeProcess_7_0_1\,\
+        com.liferay.portal.upgrade.UpgradeProcess_7_0_2
```

Comment:
We need to decide the build number of the current master here:
https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/ReleaseInfo.java#L151

Before adding a new upgrade process.

@sergiogonzalez, @adolfopa, @migue